.travis.yml

```diff
@@ -0,0 +1,39 @@
+language: python
+sudo: false
+cache: pip
+
+python:
+  - "3.6"
+  - "2.7"
+
+env:
+  - DJANGO=1.11
+  - DJANGO=2.0
+  - DJANGO=2.1
```

Comment:
I think the problem is here, see how it's done in django-netjsonconfig:
https://github.com/openwisp/django-netjsonconfig/blob/master/.travis.yml